components/widgets/inputs/users_emails_input.scss

```diff
@@ -251,13 +251,26 @@
     display: flex;
     align-items: center;
 
-    span {
-        height: 20px;
-        position: relative;
-        font-size: 16px;
-        line-height: 20px;
+    .InputErrorBox__wrapper {
         margin: auto;
-        text-align: center;
-        color: var(--error-text)
+        span {
+            height: 20px;
+            position: relative;
+            font-size: 16px;
+            line-height: 20px;
+            text-align: center;
+            color: var(--error-text)
+        }
```

Comment:
Does this css change affect any other usage of this common component?

Reply:
Are we certain that moving this span within the wrapper does not affect other places where the `users_email_input` is used?

Reply:
@pablovelezvidal wouldn't adding ```justify-content: center;``` to ```.InputErrorBox``` and then removing ```margin: auto;``` from ```.InputErrorBox > span``` do the trick?

Reply:
thanks @AGMETEOR it worked perfect!